index.js

```diff
@@ -148,6 +149,15 @@ function convertPatternProperties(schema) {
 	return schema;
 }
 
+function convertExamples(schema) {
+  if (schema['examples']) {
```

Comment:
Should there be an array check in case some pillock has invalid examples and it throws a big old error?

Reply:
Yeah, we should. I’ll add it.

Reply:
Part of 04be69f4a4e3e60cdc0ce08702a467815540aa14